src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1512,6 +1586,43 @@ private SceneDisplayer CreateEditorOrganelle()
         return node;
     }
 
+    /// <summary>
+    ///   Updates the membrane and organelle placement of the preview cell.
+    /// </summary>
+    private void UpdateCellVisualization()
+    {
+        // Don't redo the membrane when not in the preview mode to avoid possible lag spikes
+        // while editing cell structure in the structure tab
+        if (!MicrobePreviewMode || !membraneOrganellePositionsAreDirty)
+            return;
+
+        // Create a new one if not yet exist
+        if (previewMicrobe == null)
+        {
+            previewMicrobe = (Microbe)microbeScene.Instance();
+            previewMicrobe.IsForPreviewOnly = true;
+
+            // Set the initial preview cell's visibility
+            previewMicrobe.Visible = MicrobePreviewMode;
+
+            world.AddChild(previewMicrobe);
+        }
+
+        // A bit hacky but this is needed to properly visualize the edited cell completely
+        var species = (MicrobeSpecies)editedSpecies.Clone();
+        species.IsBacteria = !HasNucleus;
+        species.Colour = Colour;
```

Comment:
You already have this extra species created here, you could make the Colour change go through this rather than adding that a bit dubious override colour method in the Microbe class.